packages/application-generic/src/usecases/add-job/add-delay-job.usecase.ts

```diff
@@ -46,7 +47,13 @@ export class AddDelayJob {
         stepMetadata: data.step.metadata,
         payload: data.payload,
         overrides: data.overrides,
-        chimeraResponse: command.chimeraResponse?.outputs,
+        // TODO: Remove fallback after other delay types are implemented.
+        chimeraResponse: command.chimeraResponse?.outputs
+          ? {
+              type: DelayTypeEnum.REGULAR,
+              ...command.chimeraResponse?.outputs,
+            }
+          : undefined,
```

Comment:
Only `regular` digest type is supported currently. This change ensures that new digest types can be picked up transparently, and we can later remove this fallback.